src/+datools/+examples/+sandu/l63_experiments.m

```diff
@@ -189,213 +191,234 @@
                     'Parallel', false, ...
                     'RankHistogram', histvar, ...
                     'Tail', 'Gaussian');
-                
+
         end
-        
-        
+
+
         filter.setMean(natureODE.Y0);
         filter.scaleAnomalies(1/10);
-        
+
         mses = zeros(times - spinup, 1);
-        
+
         rmse = nan;
         ps = '';
         do_filter = true;
-        
+
         rmstempval = NaN * ones(1, times-spinup);
-        
+
         % Assimilation steps
         for i = 1:times
             % forecast/evolve the model
             nature.evolve();
-            
+
             if do_filter
                 filter.forecast();
             end
-            
+
             % observe
             xt = naturetomodel.observeWithoutError(nature.TimeSpan(1), nature.State);
             y = filter.Observation.observeWithError(model.TimeSpan(1), xt);
-            
+
             % Rank histogram (if needed)
             datools.utils.stat.RH(filter, xt);
-            
+
             % analysis
-            
+
             % try
             if do_filter
                 filter.analysis(R, y);
             end
             %catch
             %    do_enkf = false;
             %end
-            
+
             xa = filter.BestEstimate;
-            
+
             err = xt - xa;
-            
+
             if i > spinup
-                
+                % capture the rmse
                 mses(i - spinup) = mean((xa - xt).^2);
                 rmse = sqrt(mean(mses(1:(i - spinup))));
-                
+
                 rmstempval(i - spinup) = rmse;
-                
-%                 if rmse > maxallowerr || isnan(rmse) || mses(i - spinup) > 2*maxallowerr
-%                     do_enkf = false;
-%                 end
+
+                %                 if rmse > maxallowerr || isnan(rmse) || mses(i - spinup) > 2*maxallowerr
+                %                     do_enkf = false;
+                %                 end
             end
-            
+
             if ~do_filter
```

Comment:
I know I did do_enkf, but please make this dofilter or bdofilter

Reply:
Done